app/src/main/java/com/devx/mailey/presentation/core/chat/ChatFragment.kt

```diff
@@ -45,7 +45,9 @@ class ChatFragment : Fragment() {
         toastMessageListener()
         binding.sendBtn.setOnClickListener {
 
-            viewModel.sendMessage(binding.chatEditText.text.toString())
+            if(binding.chatEditText.text.toString().trim().isNotEmpty()) {
```

Comment:
In the fragments, using if expression is a bad way. It's better to use a viewModel